backends/verilog/verilog_backend.cc

```diff
@@ -2006,6 +2004,14 @@ void dump_proc_switch(std::ostream &f, std::string indent, RTLIL::SwitchRule *sw
 		}
 		f << stringf(":\n");
 		dump_case_body(f, indent + "    ", *it);
+
```

Comment:
Nit: Now that the variable is no longer used to carry state across iterations I think it's best to inline it.

Reply:
Thanks!